backend/inspirehep/records/tasks.py

```diff
@@ -135,3 +143,18 @@ def regenerate_author_records_table_entries(uuids_to_regenerate):
         record.update_authors_records_table()
         record.update_self_citations()
         db.session.commit()
+
+
+def remove_duplicate_refs_from_record(matched_inspire_record, referenced_records_in_path, path):
```

Comment:
looks much cleaner now :) Could you please also remove the extra empty line (149)?